workspaceitem-data-identifiers.md

```diff
@@ -0,0 +1,23 @@
+# WorkspaceItem data of identifiers sectionType
+[Back to the definition of the workspaceitems endpoint](workspaceitems.md)
+
+The section data represent the metadata collected for a specific [submissionform](submissionforms.md).
+It is a json object with the following structure:
+
+```json
+"identifiers" : {
+  "handle" : "http://localhost:4000/handle/123456789/6",
+  "doi" : "https://doi.org/10.5072/dspace/2",
+  "otherIdentifiers" : [ ]
```

Comment:
how the otherIdentifiers will look like? is there a chance to have an uniform representation of all the identifier types without giving a prominent role to the handle/doi?

Reply:
I wanted to at least allow for a place for other identifiers to go.
Something like the following could work:

```json
"identifiers" : [
   { "type": "handle", "value": "http://localhost:4000/handle/123456789/6" },
   { "type": "doi", "value": "https://doi.org/10.5072/dspace/2" },
]
```

And then something else could be added like 
```json
"identifiers" : [
   { "type": "handle", "value": "http://localhost:4000/handle/123456789/6" },
   { "type": "doi", "value": "https://doi.org/10.5072/dspace/2" },
   { "type": "ark", "value": "http://example.org/ark:/12025/654xz321/s3/f8.05v.tiff" },
]
```

Reply:
@kshepherd : FWIW, I'd prefer a solution that treats all identifiers as relatively "equal" if it's easy enough to do so.  In other words, your alternative option here looks good to me & I like it better than the `otherIdentifiers` option.

Reply:
I've made this change in both the submission section and the /identifiers item link data.

Reply:
Making the list extensible was a good thought.  The "otherIdentifiers" key requiring different handling was the problem IMHO.  Your suggested change works better.  (Just leaving the hash keyspace open-ended would have been "the JSON Way", I think, and would also have served.)